linux/tests/threadpool_linux_int/threadpool_linux_int.c

```diff
@@ -230,4 +276,302 @@ TEST_FUNCTION(threadpool_chaos_knight)
     execution_engine_dec_ref(execution_engine);
 }
 
+TEST_FUNCTION(one_start_timer_works_runs_once)
+{
+    // assert
+    // create an execution engine
+    volatile_atomic int32_t call_count;
+    EXECUTION_ENGINE_PARAMETERS_LINUX params;
+    params.min_thread_count = 1;
+    params.max_thread_count = 16;
+
+    EXECUTION_ENGINE_HANDLE execution_engine = execution_engine_create(&params);
+    ASSERT_IS_NOT_NULL(execution_engine);
+
+    // create the threadpool
+    THREADPOOL_HANDLE threadpool = threadpool_create(execution_engine);
+    ASSERT_IS_NOT_NULL(threadpool);
+
+    // open
+    ASSERT_ARE_EQUAL(int, 0, threadpool_open_async(threadpool, on_threadpool_open_complete, NULL));
```

Comment:
>on_threadpool_open_complete [](http://example.com/codeflow?start=63&length=27)

we are going to wait for this asynchronous function to be called or look the other way?

Reply:
here and everywhere.

Reply:
I'm removing the open, it's not needed, this is a hold over from windows.